spark/src/main/scala/org/apache/spark/sql/delta/DeltaConfig.scala

```diff
@@ -743,6 +743,17 @@ trait DeltaConfigsBase extends DeltaLogging {
     _ => true,
     "A string-to-string map of configuration properties for the managed commit owner.")
 
+  /**
+   * Maximum columnId used in the schema so far for column mapping. Internal property that cannot
+   * be set by users.
+   */
+  val MAX_AMOUNT_OF_CHAR_IN_TRANSACTION_PREDICATE = buildConfig[Int](
```

Comment:
Will also look at RFC, but the name should make it clear that this is only for debug printing. Predicates still do not have a limit